bin/start_backend.sh

```diff
@@ -56,6 +56,8 @@ done
 export STARROCKS_HOME=`cd "$curdir/.."; pwd`
 source $STARROCKS_HOME/bin/common.sh
 
+check_os_conf
```

Comment:
Just give a warning, but not exit the staring procedure?

Reply:
YES, The risk of exiting is too high and will cause frequent restarts of Supervisor